src/core/qgsmaplayer.cpp

```diff
@@ -580,9 +580,11 @@ void QgsMapLayer::writeCommonStyle( QDomElement &layerElement, QDomDocument &doc
     layerElement.appendChild( layerFlagsElem );
   }
 
-  if ( categories.testFlag( Temporal ) && const_cast< QgsMapLayer * >( this )->temporalProperties() )
+  if ( categories.testFlag( Temporal ) )
   {
-    const_cast< QgsMapLayer * >( this )->temporalProperties()->writeXml( layerElement, document, context );
+    QgsMapLayerTemporalProperties *l_temporalProperties = const_cast< QgsMapLayer * >( this )->temporalProperties();
```

Comment:
This varies somewhat from established naming conventions -- could you use lTemporalProperties instead?

Reply:
done